### PR TITLE
Add ids method and remove duplicate test

### PR DESCRIPTION
**Pull request by xuorig**
Wanted to use ids method today having ActiveRecord in mind. Thought it would be cool to add that method to frozen_record. Let me know what you think!

### CODE REVIEW

**Comment by byroot** [comment]
Interesting I wasn't aware that method even existed. I'll double check tomorrow but I don't see anything wrong with this. Thanks a lot!

File: spec/scope_spec.rb [hunk 1]
```diff
@@ -282,15 +282,6 @@
...
     end
 
-    context 'when called with multiple arguments' do
-
-      it 'returns an array of arrays' do
-        names = Country.pluck(:id, :name)
-        expect(names).to be == [[1, 'Canada'], [2, 'France'], [3, 'Austria']]
-      end
-
-    end
```
Comment (byroot) [inline spec/scope_spec.rb]: Why did you remove this test? It still has value.

File: spec/scope_spec.rb [hunk 2]
```diff
@@ -308,6 +299,28 @@
 
   end
 
+  describe '.ids' do
+
+    context 'when called with no arguments' do
+
+      it 'returns an array of ids' do
+        ids = Country.ids
+        expect(ids).to be == [1,2,3]
```
Comment (byroot) [inline spec/scope_spec.rb]: Please put spaces after commas: `[1, 2, 3]`.

**Comment by byroot** [comment]
I'll merge once the 2 comments are addressed.

File: spec/scope_spec.rb (hunk 1 above)
Comment (xuorig) [inline spec/scope_spec.rb]: it seems to be a duplicate ? or am I crazy ?

Comment (byroot) [inline spec/scope_spec.rb]: Oh indeed :facepalm:. Nevermind then.

**Comment by xuorig** [comment]
Addressed your other comment!

**Comment by byroot** [comment]
A `0.4.0` is now available on rubygems.
